CHANGELOG.md

```diff
@@ -1,8 +1,87 @@
+- Fixed crash on clicking a recovery phrase on the seed phrase verification screen. ([#4938](https://github.com/brave/brave-ios/issues/4938))
+- Fixed settings menu from staying open when a bookmark is opened in a new tab via long press on the bookmark. ([#4820](https://github.com/brave/brave-ios/issues/4820))
+
+## [1.37](https://github.com/brave/brave-ios/releases/tag/v1.37)
+
+- Implemented cosmetic filtering. ([#2683](https://github.com/brave/brave-ios/issues/2683)) 
+- Added ability to save PDF opened in Brave via share sheet menu. ([#3323](https://github.com/brave/brave-ios/issues/3323))
+- Added ability to specify custom nonce for a transaction via the settings feature in transaction confirmation panel. ([#5052](https://github.com/brave/brave-ios/issues/5052))
+- Added ability to reset local transactions from wallet settings. ([#5051](https://github.com/brave/brave-ios/issues/5051))
+- Fixed portfolio balance from showing incorrect balance when the same switching network multiple times. ([#5027](https://github.com/brave/brave-ios/issues/5027))
+
+## [1.36](https://github.com/brave/brave-ios/releases/tag/v1.36)
+
+- Added ability to add, modify and remove a custom network from wallet settings. ([#4822](https://github.com/brave/brave-ios/issues/4822))
+- Added ability to swap selected tokens directly from swap screen. ([#4979](https://github.com/brave/brave-ios/issues/4979))
+- Fixed Inline ads from showing up in Brave News when all sources are disabled. ([#4995](https://github.com/brave/brave-ios/issues/4995))
+- Fixed displaying custom token in Swap tokens list. ([#4981](https://github.com/brave/brave-ios/issues/4981))
+- Fixed displaying native currency at the top of the token list by default in Buy/Send/Swap screens. ([#4980](https://github.com/brave/brave-ios/issues/4980))
+
+## [1.35.1](https://github.com/brave/brave-ios/releases/tag/v1.35.1)
+
+- Fixed issue to show certificate error on restart when bypasses ignores the error and closes the browser. ([#4973](https://github.com/brave/brave-ios/issues/4973))
+- Fixed app crash on devices running iOS 14. ([#4967](https://github.com/brave/brave-ios/issues/4967))
+
+## [1.35](https://github.com/brave/brave-ios/releases/tag/v1.35)
+
+- Implemented ability to delete Site-Specific data. ([#4835](https://github.com/brave/brave-ios/issues/4835))
+- Deprecated Yubico SDK in favor of built-in WebAuth of WKWebView. ([#4473](https://github.com/brave/brave-ios/issues/4473))
+- Fixed websites from crashing browser in certain cases when using print functionality. ([#4848](https://github.com/brave/brave-ios/issues/4848))
+- Fixed token selection on Swap screen when opened via token details screen. ([#4677](https://github.com/brave/brave-ios/issues/4677))
+- Fixed asset detail screen to list newly added account instantly. ([#4562](https://github.com/brave/brave-ios/issues/4562))
```

Comment:
I think it only makes sense to mention `Brave Wallet` for desktop as you can use other extensions. I can update it but since mobile doesn't have any other wallet and esp iOS doesn't have anything to do with rewards Wallet would be just Brave Wallet. I can update it but just think explicitly calling out Brave Wallet everywhere isn't needed